provider/maas/environ.go

```diff
@@ -140,25 +130,13 @@ func NewEnviron(cloud environscloudspec.CloudSpec, cfg *config.Config, getCaps M
 	return env, nil
 }
 
-func (env *maasEnviron) usingMAAS2() bool {
```

Comment:
Good riddance!

Reply:
No; there was no leading API call. We discovered the version during `SetCloudSpec` and populated it on the environ.